cfn-resources/cloud-backup-restore-jobs/cmd/resource/resource.go

```diff
@@ -228,13 +228,14 @@ func List(req handler.Request, prevModel *Model, currentModel *Model) (handler.P
 
 	models := make([]interface{}, 0)
 	if *currentModel.InstanceType == serverlessInstanceType {
-		serverless, resp, err := client.Atlas20231115002.CloudBackupsApi.ListServerlessBackupRestoreJobs(context.Background(), *currentModel.ProjectId, *currentModel.InstanceName).Execute()
+		serverless, resp, err := client.AtlasSDK.CloudBackupsApi.ListServerlessBackupRestoreJobs(context.Background(), *currentModel.ProjectId, *currentModel.InstanceName).Execute()
```

Comment:
change the type of the `models := make([]interface{}, 0)` ?

Reply:
The property `ResourceModels` in `ProgressEvent` struct is actually typed as `[]interface{}` so this type definition cannot be adjusted to a more specific type (can see we do the same in all resources).